.github/workflows/build-kernel-arcvm.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
           sudo ln -s --force python3 /usr/bin/python
 
-          export LLVM_VERSION=12
+          export LLVM_VERSION=18
```

Comment:
You should not change the version of the compilation toolchain, whether it is AOSP or kernel source code, there are requirements for the compiler; AOSP even only allows you to use a fixed version of Ubuntu. No one is willing to fight with the compiler for no reason.